aea/cli/remove.py

```diff
@@ -281,6 +271,9 @@ def remove_unused_component_configurations(ctx: Context) -> Generator:
 
     Context manager!
     Clean all configurations on enter, restore actual configurations and dump agent config.
+
+    :param ctx: click context
+    :yield: generator
```

Comment:
This should be "yield None", although since we chose to ignore "return None" we should ignore this as well... what does Darglint thinks about this?
```suggestion
    :yield: None
```